drivers/sensor/bosch/bmp180/Kconfig

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2024 Chris Ruehl
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig BMP180
+	bool "Bosch BMP180 pressure sensor"
+	default y
+	depends on DT_HAS_BOSCH_BMP180_ENABLED
+	select I2C if $(dt_compat_on_bus,$(DT_COMPAT_BOSCH_BMP180),i2c)
+	help
+	  Enable driver for the Bosch BMP180 pressure sensor
+
+if BMP180
+
+config BMP180_OSR_RUNTIME
```

Comment:
Like to stuck to the Kconfig solution, but change the default to 'no'.  If someone decide to use the oversampling updates in her/his application it will spit an error when not set to CONFIG_BMP180_OSR_RUNTIME=y in the prj.conf. And we shrink the binary size a tiny little bit.